rustainers/src/runner/inner.rs

```diff
@@ -291,7 +291,14 @@ pub(crate) trait InnerRunner: Display + Debug + Send + Sync {
                         "{scheme}://127.0.0.1:{host_port}/{}",
                         path.trim_start_matches('/')
                     );
-                    let Ok(response) = reqwest::get(&url).await else {
+                    let Ok(client) = reqwest::ClientBuilder::new()
+                        .danger_accept_invalid_certs(true)
```

Comment:
That's too dangerous, I understand the use case, so I think we should allow the use to choose if this value.

Reply:
Excellent feedback - exactly what I was looking for!

> document to explain that for testing purpose the default value is allowing invalid certificate.

If the option is there I'm okay with having `require_valid_certs: true` as the default (I'd just set it to `false` in my code).

I assume you're talking about `WaitStrategy::https` function?